### ISSUE TITLE
Show apps button not responding to touch with x11

### ISSUE THREAD

**Issue by SKEdmis** [home-sweet-gnome/dash-to-panel#1276]
When using Wayland the show apps button responds to touch as expected. With x11 it doesn't respond to touch. It does respond to mouse clicks and super+a. I see this behavior in Manjaro Gnome 20.2 and Fedora 33. I'm using D to P 40 and Gnome 3.38.3 on both distros. I installed the extension from the Gnome extensions website.
Edit: Show apps button works on Gnome dock when D to P is disabled.

**Comment by SKEdmis** [comment]
An oversight on my part: Dash to Panel is a system installed extension in Manjaro Gnome. I did not install it from the extensions website. I did install it from the extensions website on Fedora, however. For what it's worth, I uninstalled Dash to Panel in Pamac, rebooted and then reinstalled. The show apps button is still unresponsive to touch with x11. I have issues when running Wayland so switching to Wayland is not currently a possibility for me. I noticed, too, that the show apps button is responsive to touch if I first hit the super key and then touch the button. I know this is a small issue for most. If you'd like to remove this thread that's fine with me.
